internal/gui/gui.go

```diff
@@ -54,6 +55,38 @@ func New(db *db.Database) *Gui {
 	}
 }
 
+func (g *Gui) ProcessColors(colors map[string]string) {
```

Comment:
I only want to comment here that there might be a better way to do this. 

Also is it more efficient (in Go) to do what I did checking `hex == ""` or would it be better to wrap the `switch` with an `if` statement with the following conditional: `hex != ""`?